utils/page_objects/signup_page/signup_page.js

```diff
@@ -0,0 +1,129 @@
+            },
+        };
+
+        this.firstName = {
+            locatorType: "name",
+            locatorId: "firstName",
+        };
+
+        this.lastName = {
+            locatorType: "name",
+            locatorId: "lastName",
+        };
+
+        this.postalCodeNld = {
+            locatorType: "name",
+            locatorId: "address.postalCodeNld",
+        };
+
+        this.houseNumber = {
+            locatorType: "name",
+            locatorId: "address.houseNumber",
+        };
+
+        this.houseNumberExtra = {
+            locatorType: "name",
+            locatorId: "address.houseNumberExtra",
+        };
+
+        this.emailAddress = {
+            locatorType: "name",
+            locatorId: "emailAddress",
+        };
+
+        this.password = {
+            locatorType: "name",
+            locatorId: "password",
+        };
+
+        this.phoneNumberNl = {
+            locatorType: "css",
+            locatorId: "#phoneNumber",
+        };
+
+        this.birthDay = {
+            locatorType: "css",
+            locatorId: "#dateOfBirthWebshop",
+        };
+        this.radioOptOut = {
+            locatorType: "id",
+            locatorId: "f-radio-button-bonusCardChoice-optOut",
+            span: {
+                locatorType: "xpath",
+                locatorId: '//*[@id="app"]/div/main/div/div/div/form/div[7]/div[1]/div/span[3]/label',
+            },
+        };
+
+        this.radioRequest = {
+            locatorType: "id",
+            locatorId: "f-radio-button-bonusCardChoice-request",
+            span: {
+                locatorType: "xpath",
+                locatorId: '//*[@id="app"]/div/main/div/div/div/form/div[7]/div[1]/div/span[2]/label',
+            },
+        };
+
+        this.radioInput = {
+            locatorType: "id",
+            locatorId: "f-radio-button-bonusCardChoice-input",
+            span: {
+                locatorType: "xpath",
+                locatorId: '//*[@id="app"]/div/main/div/div/div/form/div[7]/div[1]/div/span[1]/label',
+            },
+        };
+
+        this.bonusCardNumberField = {
+            locatorType: "xpath",
+            locatorId: '//*[@id="cards.BO"]',
+        };
+
+        this.checkboxServiceMail = {
+            locatorType: "id",
+            locatorId: "f-checkbox-serviceMail",
+            span: {
+                locatorType: "xpath",
+                locatorId: '//*[@id="app"]/div/main/div/div/div/form/div[8]/div/label',
+            },
+        };
+
+        this.checkboxMail = {
+            locatorType: "id",
+            locatorId: "f-checkbox-newsLetter",
+            span: {
+                locatorType: "xpath",
+                locatorId: '//*[@id="app"]/div/main/div/div/div/form/div[9]/div/label',
+            },
+        };
+
+        this.registrationSubmitButton = {
+            locatorType: "id",
+            locatorId: "registration-form-submit",
```

Comment:
Please refer to https://github.com/axferox/Task_7_WebdriverJS_AH/tree/own-variant to see how the concept of element getters is usually implemented in the PageObject pattern.